integration/hive/src/main/java/org/apache/carbondata/hive/MapredCarbonInputFormat.java

```diff
@@ -106,9 +107,17 @@ private static CarbonTable getCarbonTable(Configuration configuration, String pa
     CarbonInputSplit split;
     for (int i = 0; i < splitList.size(); i++) {
       split = (CarbonInputSplit) splitList.get(i);
-      splits[i] = new CarbonHiveInputSplit(split.getSegmentId(), split.getPath(), split.getStart(),
-          split.getLength(), split.getLocations(), split.getNumberOfBlocklets(), split.getVersion(),
-          split.getBlockStorageIdMap());
+      CarbonHiveInputSplit inputSplit = new CarbonHiveInputSplit(split.getSegmentId(),
+              split.getPath(), split.getStart(), split.getLength(),
+              split.getLocations(), split.getNumberOfBlocklets(),
+              split.getVersion(), split.getBlockStorageIdMap());
+      BlockletDetailInfo info = new BlockletDetailInfo();
+      info.setBlockSize(split.getLength());
+      info.setBlockFooterOffset(split.getDetailInfo().getBlockFooterOffset());
+      info.setVersionNumber(split.getVersion().number());
+      info.setUseMinMaxForPruning(false);
```

Comment:
Why do you set false in here?

Reply:
There is no need to use min max for pruning.